test-files/old-tests/param-direction.cpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * @param x
+**/
+void f(int x) { }
+
+/** test brief
+ * @param x
+ * @param[in] y
+**/
+void g(int x, int y) { }
+
+/**
+ * @param x
+ * @param[in,out] y
```

Comment:
Doesn't clang handle `@param[inout]` ?

Reply:
@vinniefalco @klemens-morgenstern it does not